elan/rpc/eclient.go

```diff
@@ -56,11 +56,13 @@ func (e *elanClient) UpdateActionResult(req *pb.UpdateActionResultRequest) (*pb.
 	return e.s.UpdateActionResult(ctx, req)
 }
 
-func (e *elanClient) UploadIfMissing(entries []*uploadinfo.Entry) error {
+func (e *elanClient) UploadIfMissing(entries []*uploadinfo.Entry, compressors []pb.Compressor_Value) error {
```

Comment:
Does changing this signature mean that all the Please clients will fail for a short period of time if we roll Mettle first?

Reply:
Not this particular case, no. This interface is only relevant within the process.

The upload/download blob stuff should all be compatible here, the main bit that's not is the `Capabilities` stuff (which I think we can work around by ordering the rollout; we could probably avoid it by doing it in several more steps but I don't think it's worth it)